libraries/lwrp.rb

```diff
@@ -15,7 +15,7 @@ module CernerSplunk
   # Methods involved with augmenting the LWRP syntax / writing recipies
   module LWRP
     # Change a list of monitors to a hash of stanzas for writing to a config file
-    def self.convert_monitors(node, monitors, default_index = nil, base = {})
```

Comment:
Bikeshedding: on one hand, what if we kept the method signature as is for now just didn't call the other method for passivity, But also this should likely only be an internal detail so, changing the signature and updating everywhere used should still be a non-major release for changing the signature